recipes/bareditor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "bareditor" %}
+{% set version = "0.17" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6dbf897ff03a75182c23c3367dc110bf676857df9548e4bca6b0b452fdf0f432" %}
```

Comment:
```suggestion
```

Reply:
Can it be parametrized such as `{% set sha256 = "..." %}` and be used as `sha256: {{ sha256 }}`? I had a discussion about its convenience before in https://github.com/conda-forge/staged-recipes/pull/9739/files#r331470797, and was told it's OK to keep it as long as the name does not change. Having the name, version and sha256 in the beginning of the file makes maintenance more convenient.

Reply:
if you think so, ok.

Reply:
Sorry for the delay, lost a moment to address it on Sunday. 6b6db2e addresses it.